src/gt4py/backend/module_generator.py

```diff
@@ -465,11 +465,15 @@ def _is_not_empty(self) -> bool:
     def generate_imports(self) -> str:
         source = ["from gt4py import utils as gt_utils"]
         if self._is_not_empty():
+            assert self.pyext_file_path is not None
+            file_path = "f{{pathlib.Path(__file__).parent.resolve()}}/{}".format(
+                os.path.basename(self.pyext_file_path)
+            )
```

Comment:
It looks like there is an error here. Should this be something like:
```suggestion
            file_path = "{{pathlib.Path(__file__).parent.resolve()}}/{base}".format(
                base=os.path.basename(self.pyext_file_path)
            )
```
?

Reply:
The `f` has to be pulled outside the quotes. I think I got it now. The latest commit I pushed works locally.